nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.github/workflows/ci.yml

```
@@ -0,0 +1,20 @@
+name: nf-core CI
+# This workflow is triggered on pushes and PRs to the repository.
+on: [push, pull_request]  
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        nxf_ver: ['19.04.0', '']
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install Nextflow
+        run: |
+          export NXF_VER=${{ matrix.nxf_ver }}
```

This is causing the tests to fail, because the double brackets look like a jinja variable.
```suggestion
          {% raw %}export NXF_VER=${{ matrix.nxf_ver }}{% endraw %}
```

---

Should we merge this into a target branch e.g. `nf-core/tools:github-actions` so that we can add stuff missing? I have some additional checks for PRs coming from `patch` branches which are missing here too but can't commit to Maximes branch 👍